### ISSUE TITLE
pip_wrapper, impossible to write to stderr

### ISSUE THREAD

**Issue by matthieumarrast** [metwork-framework/mfext#1513]
using `pip` command raises multiple "Permission denied" errors while trying to write to `/dev/stderr`

```
[mfserv:~/my_plugin]↥ develop ± pip
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 10: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 11: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 12: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 13: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 14: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 15: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 16: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 17: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 18: /dev/stderr: Permission denied
/opt/metwork-mfext-2.1/bin/pip_wrapper: line 19: /dev/stderr: Permission denied
```

solution: write to file in module tmp or module log, or write to user stdout...

**Comment by thebaptiste** [comment]
A known sudo/bash issue : https://superuser.com/questions/1699317/permission-denied-on-dev-stderr-after-sudo

**Comment by thebaptiste** [comment]
Is there a good reason for us to do `echo xxxx >>/dev/stderr` rather than `echo xxxx >>/dev/stdout ` or simple `echo xxxx` in this shell ?

**Comment by thebaptiste** [comment]
Or maybe we can add `chmod a+w /dev/stderr` when installing mfext. It should solve the problem.

**Comment by matthieumarrast** [comment]
Or we can simply remove all these `>>/dev/stderr` so messages will be displayed to the user output. It will results to :
```
± pip
ERROR: you should only use pip when:

- you are a standard python virtualenv
- you are in a plugin_env (go to your plugin directory and use 'plugin_env'
  command to get into the 'plugin_env'


=> If you are REALLY SURE about what you are doing, you can override this
by setting ALLOWPIP=1 env var before executing this command
(but there is probably NO GOOD REASON TO DO THAT if you are not a MetWork guru)
```

**Comment by thebaptiste** [comment]
Ok, the simpler the better.
By the way, the problem seems to be the same with `/dev/stdout` after `su -`

**Comment by thebaptiste** [comment]
Closed by #1530
